global.yaml

```
@@ -72,6 +72,7 @@ global:
   # Whether or not to use spire for cert management and the trust domain
   spire:
     enabled: true
+    image_pull_secret: docker.secret
```

Let's make this a map of values incase we need to provide more than a single image pull secret:

```suggestion
    image_pull_secret: 
      - name: docker.secret
      - name: imagepullSecret2
```

---

Changed the format of the Spire image_pull_secrets to be a list.